signing/src/test/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlSignerParserTest.java

```diff
@@ -73,6 +75,11 @@ class YamlSignerParserTest {
 
   private YamlSignerParser signerParser;
 
+  @BeforeAll
+  static void init() {
+    YamlMapperProvider.INSTANCE.init(Optional.empty());
```

Comment:
Can't we just create a test fixture version of the YamlMapper and use it everywhere?

Reply:
The requirement of explicit init is removed as getInstance().getMapper() would initialize it with defaults if not already initialised.